### PR TITLE
Feature 1

### PR DESCRIPTION
**Pull request by Qoosim**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      </nav>
+      <div class="your-verse">
+        <img src="images/your_verse_title_small.png" alt="" />
+        <img src="images/your_verse_subhead_small.png" alt="" id="subhead" />
+        <div class="film-story">
+
+        </div>
+      </div>
+    </main>
+    <div class="wrapper">
```
Comment (samgaco) [inline index.html]: The four boxes inside the container with the "wrapper" class should be linked. 

(the `<div>` could be contained within anchor elements with a # pound to be linked)

**Review by samgaco — CHANGES_REQUESTED** [review]
#### Hi @Qoosim :high_brightness: 

#### Is looking really good :clap:  some things to address before we can give you the approval:

1.  **[REQUIERED]** The four boxes inside the container with the "wrapper" class should be linked.
2.  **[REQUIERED]** A horizontal scrolling bar is showing, you should find what is causing it and correct it.
3.  **[REQUIERED]** Please read about [HTML semantic ](https://www.w3schools.com/html/html5_semantic_elements.asp)elements and implement them in your project. For example `<article>` , `<section>` ....
4. **[REQUIERED]** You should fix the errors given to you by the[ validator ](https://validator.w3.org)when uploading the index file.

----------------

* **[OPTIONAL]**  Improve the readme file. Find some inspiration [here](https://github.com/othneildrew/Best-README-Template).

----------------
Moving on have this into account :arrow_down: 

* **[For your next code review requests]**  Use meaningful names for your Pull Requests
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

--------------------

_Summary: the points (1,2,3,4) are required and you have the option of working on the readme file. Work on these changes and submit back for another code review once you are done_ :muscle: 

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

**Comment by samgaco** [comment]
reviewing

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+By: Qoosim AbdulGhaniyy Adekunle
+
+
+This project consists of building an apple website homepage using background
+image and adding gradient to elements. The goal is to make a clone of an old
+version of apple website homepage.
+
+For more information about the project. Check The Odin Project website ["https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients"]
+
+For the live version of the project, check here: ["https://raw.githack.com/Qoosim/apple-old-version/feature-1/index.html"]
```
Comment (samgaco) [inline README.md:14]: You should resolve the conflicts first, they seem to come from the read me file, so maybe pushing the same changes in this branch to the master branch before merging would help you.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+          <a href="#"><img src="images/promo_iphone5s.jpg" alt="iPhone 5s" /></a>
+        </div>
+        <div class="inner-wrapper">
+          <a href="#"><img src="images/promo_iphone5c.jpg" alt="iPhone 5c" /></a>
+        </div>
+        <div class="inner-wrapper">
+          <a href="#"><img src="images/promo_30_years.jpg" alt="iPhone promo year" /></a>
+        </div>
+        <div class="inner-wrapper">
+          <a href="#"><img src="images/promo_supplier_responsibility.jpg" alt="iPhone Supplier" /></a>
```
Comment (samgaco) [inline index.html:51]: Good :+1: another suggested way of going about this section which could be also be `<aside>` would be too set the images as backgrounds of the container and wrap the `<div>` within `<a>` tags.

**Review by samgaco — APPROVED** [review]
### Hi @Qoosim :hand: 

#### Everything is perfect now. You may now merge to the master branch and submit the completion form. :muscle: 

##### Just bear in mind that you should resolve the conflicts first, they seem to come from the read me file, so maybe pushing the same changes in this branch to the master branch before merging would help you.

--------------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Comment by Qoosim** [comment]
Thanks for the suggested way. I tried it and it was better and awesome.
More wisdom comrade

**Pull request merged**